src/html/index.html

```diff
@@ -76,12 +77,9 @@
 	<link href="app:/_/theme-superhero.css" type="text/css" data-theme="superhero"/>
 <head>
 <body>
-	<script type="text/javascript">
+	<!-- substitute:main -->
 
-		$.noConflict();
```

Comment:
$ is for jQuery

$.noConflict() removes $ from the global namespace

http://api.jquery.com/jQuery.noConflict/